### DISCUSSION TITLE
Feature Request: .Net Framework And Java VM Backend.

### DISCUSSION THREAD

**Discussion by KeitoTobi1** [vlang/v#11617] [Ideas]

**Comment by Metalymph** [comment]
I think JVM backend can be bypassed having LLVM backend (already proposed). In fact with GraalVM you can use `lli` modules with Java/Kotlin/Nodejs and so on... About .NET, you mean .NET Core?

**Comment by KeitoTobi1** [comment]
> About .NET, you mean .NET Core?

This Is Both.

**Comment by nedpals** [comment]
I remembered not too long ago that someone has forked the repository and is in the process of porting it into .NET for enterprise use but don't know what's the issue number and the progress

**Comment by KeitoTobi1** [comment]
Java VM Compile Optimized Java Programming Language.
.Net Framework Compile Optimized C# Programming Language.
